proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -363,6 +363,9 @@ message MsgCreateConsumer {
 
   // allowlisted reward denoms of the consumer
   AllowlistedRewardDenoms allowlisted_reward_denoms = 6;
+
+  // infraction parameters for slashing and jailing
+  InfractionParameters infraction_parameters = 7;
```

Comment:
Is this optional when the Tx is submitted via CLI/RPC?

It seems very annoying to think about such details during MsgCreateConsumer.

Reply:
yes, its optional. if its not set, default values(same as the provider values) will be set.